app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -235,18 +256,47 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
       $scope.emsCommonModel.ssh_keypair_auth_status         = true;
       $scope.emsCommonModel.metrics_auth_status             = data.metrics_auth_status;
       $scope.emsCommonModel.vmware_cloud_api_version        = '9.0';
+      $scope.emsCommonModel.virtualization_selection        = data.virtualization_selection;
+      $scope.emsCommonModel.kubevirt_api_port               = $scope.emsCommonModel.default_api_port;
+      $scope.emsCommonModel.kubevirt_auth_status            = data.kubevirt_auth_status;
+      $scope.emsCommonModel.kubevirt_security_protocol      = $scope.emsCommonModel.default_security_protocol;
+      $scope.emsCommonModel.kubevirt_tls_ca_certs           = $scope.emsCommonModel.default_tls_ca_certs;
+      $scope.emsCommonModel.kubevirt_password               = data.kubevirt_password;
 
       miqService.sparkleOff();
-
+      $scope.hideDisabledTabs();
       $scope.afterGet  = true;
       $scope.modelCopy = angular.copy( $scope.emsCommonModel );
     }
   };
 
+  $scope.hideDisabledTabs = function() {
```

Comment:
@AparnaKarve this is how I managed to hide the tabs if they should be disabled.

**Add Kubernetes**
![Add Kubernetes](https://user-images.githubusercontent.com/316242/35102403-9139012e-fc6b-11e7-89b2-816ac6197c2c.png)

**Add Kubernetes with virtualization**
![selection_186](https://user-images.githubusercontent.com/316242/35102409-946f2594-fc6b-11e7-8e12-4b2a3401c33a.png)

**Edit Kubernetes with virtualization**
![selection_187](https://user-images.githubusercontent.com/316242/35102529-db2b1218-fc6b-11e7-909c-ba80829095ae.png)

Please let me know if this fix requires additional work.